### DISCUSSION TITLE
Where is the v5 documentation?

### DISCUSSION THREAD

**Discussion by Lani** [mobxjs/mobx#2484] [Q&A] [answered]
Hello!

I must be blind, but I cannot find the v5 documentation anywhere? 

Links in the readme in the mobx4and5 branch no longer seems to work. I can see the docs in the docs folder, but it is not as easy to browse as the website.

**Comment by danielkcz** [comment, marked as answer]
You are right it wasn't preserved anywhere, unfortunately. However, most core concepts are still the same. All changes are pretty much summarized in [the blog post](https://michel.codes/blogs/mobx6) from Michel. You can just ignore those and the rest of the current docs site is still relevant.

Also, note, there is no longer any support for V4/V5 and it's highly recommended to migrate.

**Reply by Lani** [reply]
Ah ok, thank you for your reply. We haven't had time to upgrade to v6 yet 😅 

Also, thank you for all the excellent work on MobX, including v6, looks even more awesome!
